tests/test_utils.py

```diff
@@ -22,3 +22,20 @@ def test_build_workspace_path():
     assert build_workspace_path(0) == os.path.join(
         SHARED_VOLUME_PATH, "users/0/workflows"
     )
+
+    assert build_workspace_path(0, workflow_id=1) == os.path.join(
```

Comment:
What about calling this function passing the `user_id` arg name to improve readability?

```
build_workspace_path(user_id=0, workflow_id=1)
```